### DISCUSSION TITLE
How to merge KeeShare files?

### DISCUSSION THREAD

**Discussion by neopostmodern** [keepassxreboot/keepassxc#10720] [Q&A] [answered]
I share a password group through KeeShare and then sync the file through NextCloud. Apparently simultaneous edits happened and NextCloud created conflict files, which means there are now multiple versions of the same file with different edits applied. How can I merge these KeeShare files?
(I'm aware of the functionality to merge the main database, but in this case it's a separate KeeShare file. Maybe it would work, but I'm afraid to try given there some funky bugs with KeeShare data being deleted.)

**Comment by droidmonkey** [comment, marked as answer]
open the keeshare file independently and then do a "Merge from database"

**Reply by neopostmodern** [reply]
Great, that worked! Thank you. 

For future readers: the password to open the KeeShare file independently is the one from the group/folder's KeeShare settings.
